srcpkgs/NetworkManager/patches/fix-cross-gir.patch

```diff
@@ -1,52 +1,29 @@
 Reason: Fix crossbuilding GIR
 Upstream: No, not upstreamable since this is very specific to the Void setup
-diff --git clients/common/meson.build clients/common/meson.build
-index afa257b..50b53e7 100644
---- clients/common/meson.build
-+++ clients/common/meson.build
-@@ -35,7 +35,7 @@ if enable_introspection
-     settings_docs,
-     input: nm_property_docs,
-     output: settings_docs,
--    command: [xsltproc, '--output', '@OUTPUT@', join_paths(meson.current_source_dir(), 'settings-docs.xsl'), '@INPUT@'],
-+    command: [xsltproc, '--output', '@OUTPUT@', join_paths(meson.current_source_dir(), 'settings-docs.xsl'), join_paths(meson.current_source_dir(), '..', '..', '@INPUT@')],
-   )
- 
-   test(
-diff --git libnm/meson.build libnm/meson.build
-index 51ca46d..f14d861 100644
---- libnm/meson.build
-+++ libnm/meson.build
-@@ -236,7 +236,13 @@ if enable_introspection
-     )
-   endif
- 
--  generate_setting_docs = join_paths(meson.current_source_dir(), 'generate-setting-docs.py')
+--- libnm/meson.build.orig	2020-12-28 16:03:30.641850261 +0100
++++ libnm/meson.build	2020-12-28 16:31:54.932136481 +0100
+@@ -276,6 +276,14 @@
+     'GI_TYPELIB_PATH=' + gi_typelib_path,
+     'LD_LIBRARY_PATH=' + ld_library_path,
+   ]
++  
 +  if meson.is_cross_build()
 +    qemu = '/usr/bin/qemu-@XBPS_TARGET_QEMU_MACHINE@-static'
-+    qemu_python3_cmd = qemu + ' -L @XBPS_CROSS_BASE@ @XBPS_CROSS_BASE@/usr/bin/python3'
-+    generate_setting_docs = ['sh', '-c', qemu_python3_cmd, join_paths(meson.current_source_dir(), 'generate-setting-docs.py')]
++    qemu_python3_cmd = qemu + ' -L @XBPS_CROSS_BASE@ @XBPS_CROSS_BASE@/bin/python3'
++    generate_setting_docs = ['sh', '-c', qemu_python3_cmd, join_paths(meson.source_root(), 'tools', 'generate-docs-nm-settings-docs-gir.py')]
```

Comment:
I'm just a tad confused because I believe our cross base directories are missing the `/bin -> /usr/bin` symlink, so switching to `XBPS_CROSS_BASE/bin/python3` shouldn't exactly be working...

Reply:
I have no idea why it works. The fix-cross-gir patch I've submitted here is an altered version of the one being worked on in the PR linked above with all of the sections dealing with deleted or changed files taken out.